src/porofluid_pressure_based_elast_scatra/4C_porofluid_pressure_based_elast_scatra_artery_coupling_pair.cpp

```diff
@@ -2703,6 +2718,22 @@ void PoroPressureBased::PorofluidElastScatraArteryCouplingPair<dis_type_artery,
   }
 }
 
+/*----------------------------------------------------------------------*
+ *----------------------------------------------------------------------*/
+template <Core::FE::CellType dis_type_artery, Core::FE::CellType dis_type_homogenized, int dim>
+double PoroPressureBased::PorofluidElastScatraArteryCouplingPair<dis_type_artery,
+    dis_type_homogenized, dim>::evaluate_artery_flow_rate(const double artery_pressure_gradient)
+    const
+{
+  const double radius = artery_diameter_at_gp_ / 2.0;
+
+  // Hagen-Poiseuille equation
+  const double flow_rate =
+      -artery_pressure_gradient * M_PI * pow(radius, 4) / (8.0 * artery_material_->viscosity());
```

Comment:
```suggestion
      -artery_pressure_gradient * M_PI * radius* radius * radius * radius / (8.0 * artery_material_->viscosity());
```

minor: an integer power is a super simple operation, but going through `std::pow` will use a general algorithm for float exponents.